### PR TITLE
[MVI-1135] (requests-mv-integrations) Integrate smart-cast

### PR DESCRIPTION
**Pull request by StasTune**
Move to using safe-cast package for casting data types

* Add safe-cast package 0.0.8 to requirements
* Update all casting function imports to happen from safe-cast
* Delete safe-cast.py from support folder and its init.py
* Update package version to 0.4.3 (Including History.rst)

### CODE REVIEW

**Comment by jefftune** [comment]
Travis CI is currently breaking:
`No matching distribution found for safe_cast>=0.0.8 (from requests-mv-integrations==0.4.3)`

File: requirements.txt [hunk 1]
```diff
@@ -13,6 +13,7 @@ python-dateutil==2.6.0
 requests-toolbelt==0.7.0
 requests==2.13.0
 requests-mock==1.3.0
+safe_cast>=0.0.8
```
Comment (jefftune) [inline requirements.txt]: I think this should be relabeled as `safe-cast>=0.0.8`

**Comment by StasTune** [comment]
The TravisCI fail is not related to the changes in safe_cast. 
The requirement file has been updated.

**Comment by StasTune** [comment]
Bug in master fixed and so this PR is passing tests as well.

**Pull request merged**